test_cases/process_test-kcs-onCreate-process.js

```diff
@@ -9,7 +9,13 @@ export default (folderName, Common) => {
       it('Process inserts a record after inserting a label value on the tab "process > onCreate"', () => {
          cy.get(
             '[data-cy="string label 06a93149-590d-4e4f-9463-5ff43a689fd1 2172ba78-b327-42a1-8918-d97852234aee"]',
-         ).type("test label");
+         ).type("test label", { delay: 0 });
+
+         cy.wait(1000);
```

Comment:
I don't like this. Did you try like this:
```js
cy.get(selector).as("myField").type("anything").clear();
cy.get("@myField").type("test label")
```

Reply:
OK, so I didn't actually try the `.type().clear()` approach.  I can try that if you like.

What I'm noticing is that we have a new event being issued on the UI that is causing the Form / Textbox to clear again.  On my system, I'm seeing that if I simply wait, the clear will happen and then I can continue on.  Otherwise I'm not sure how to detect the refresh before we attempt to type into the field.

Trying the approach, I'm suspicious that my system will simply zip through the process, and the UI reset will still happen during or after the second `.type()`.  That will reset the value and the test still fails.

Reply:
> What I'm noticing is that we have a new event being issued on the UI that is causing the Form / Textbox to clear again.
Do we want this? Maybe we want to check that.

>I can try that if you like.
Yes, please try. If it doesn't work we can go back to wait

Reply:
ok, take that back.  The `.type().clear()` approach seems to work well locally.